qutebrowser/config/configfiles.py

```diff
@@ -94,7 +95,10 @@ def init_save_manager(self, save_manager):
         save_manager.add_saveable('yaml-config', self._save)
 
     def _save(self):
-        """Save the changed settings to the YAML file."""
+        """Save the settings to the YAML file if they've changed."""
+        if self.values == self._initial_values:
+            return
```

Comment:
Can you please take a look at `tests/unit/config/test_configfiles.py` and add a test for this scenario?